rallyrolebot/cogs/__init__.py

```diff
@@ -1 +1,8 @@
-from cogs import role_cog, update_cog, rally_cog, channel_cog
+from cogs import (
```

Comment:
TODO: automatically import all python files into __all__ so that changes to this file is unnecessary,